client-dotnet/src/EvoMaster.Controller/Controllers/db/DbCleaner.cs

```diff
@@ -171,11 +171,15 @@ public static class DbCleaner {
             command.ExecuteNonQuery();
         }
 
-        private static void DeleteTables(DbCommand command, string table, ISet<string> tableHasIdentify) {
-            command.CommandText = "DELETE FROM " + table;
+        private static void DeleteTables(DbCommand command, string table, string schmea, ISet<string> tableHasIdentify){
+            var tableWithSchema = table;
+            if (schmea.Length > 0 && !schmea.Equals(GetSchema(DatabaseType.MS_SQL_SERVER))){
+                tableWithSchema = schmea + "." + table;
```

Comment:
add bit explanation. is it that MSSQL always want the schema name specified?

Reply:
but this is not a test case, isn't it? so it is still unclear why MS_SQL_SERVER is treated specially here

Reply:
I thought the comment is for DbClearner tests. now, the comment is added in the latest commit.